include/fieldperp.hxx

```diff
@@ -80,6 +80,8 @@ class FieldPerp : public Field {
   const DataIterator begin() const;
   const DataIterator end() const;
 
+  const IndexRange region(REGION rgn) const;
```

Comment:
This seems separate from the addition of regions to min/max and doesn't seem to be used, is this in preparation for something else?

Reply:
This is unrelated.
It allows to iterate over a fieldperp - but I don't think I use it in the aiolos mesh.
It is rather to unify the fields.